### PR TITLE
Update parsing of Kamereon vehicle details

### PR DESCRIPTION
**Pull request by epenet**
Add parsing of vehicle details:
- get_registration_number
- get_energy_code
- get_brand_label
- get_model_label

### CODE REVIEW

File: tests/fixtures/kamereon/Contributions.rst [hunk 1]
```diff
@@ -0,0 +1,27 @@
+Contributing
```
Comment (Quentame) [inline tests/fixtures/kamereon/Contributions.rst:1]: CONTRIBUTING file

File: src/renault_api/model/kamereon.py [hunk 2]
```diff
@@ -92,18 +92,95 @@ class KamereonPersonResponse(KamereonResponse):
 )()
 
 
+class EnergyCode(Enum):
+    """Enum for vehicleDetails energy code."""
+
+    ESSENCE = "ESS"
+    ELECTRIQUE = "ELEC"
```
Comment (Quentame) [inline src/renault_api/model/kamereon.py:99]: In english

**Review by Quentame — COMMENTED** [review]
I don't think always checking `None` is usefull

File: tests/fixtures/kamereon/Contributions.rst (hunk 1 above)
Comment (epenet) [reply tests/fixtures/kamereon/Contributions.rst:1]: I have created PR #71 to address this comment.

But where do you recommend that we add comments about the json files that people have already provided?

File: src/renault_api/model/kamereon.py (hunk 2 above)
Comment (epenet) [reply src/renault_api/model/kamereon.py:99]: Addressed in PR #68 : https://github.com/hacf-fr/renault-api/pull/68/commits/28c27b9028158d3c5be1ccc7d9259481ef7be6c0

**Comment by epenet** [comment]
Regarding checking for `None` => Addressed in PR #68 : https://github.com/hacf-fr/renault-api/pull/68/commits/68b32960dd37f3f8dabcb8139b91595dba78a892

**Pull request merged**